src/components/NewsRoomCard.jsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  Button
+} from "@baltimorecounty/dotgov-components";
+import React from "react";
+
+const NewsRoomCard = props => {
+  const { publishDate, title, author, articleSummary, thumbnail, url } = props;
+
+  const options = {
+    year: "numeric",
+    month: "long",
+    day: "numeric"
+  };
+
+  const published = new Date(publishDate).toLocaleDateString("en-US", options);
+
+  return (
+    <Card>
+      <CardContent>
+        <div style={{ textAlign: "left" }}>
+          <h3>{title}</h3>
+          <div className="itemNewsRoom">
+            <div className="itemMiddle itemMiddleWidth">
+              <i className={thumbnail ? thumbnail : "fas fa-newspaper"} />
+            </div>
+            <div className="itemMiddle">
+              <div className="itemLink">
+                <div
+                  style={{
+                    borderRight: " 2px solid gray",
+                    paddingRight: "20px",
+                    marginRight: "20px"
+                  }}
+                >
+                  <a href={url}>{published} </a>
```

Comment:
these are not links, I know it shows in xd, but there are user stories for these items, that may or may not be completed. When i'm in the office I'll take a look and see if this is clear in the acceptance criteria. If not, this is something we should bring up in the retrospective.

Reply:
Changed to <p> and added styles to match XD. I dont see where these should have been anchors. Saw blue underlined and assumed. So if they are supposed to anchors later we can change them back.

Reply:
yes @tmccoy529 sam basically said this isn't important. There are some usability concerns with making these filters right away as well.

Reply:
yes @tmccoy529 sam basically said this isn't important. There are some usability concerns with making these filters right away as well.